src/ServiceControl/MessageFailures/FailedMessagesFeature.cs

```diff
@@ -18,13 +18,14 @@ public class FailedMessagesFeature : Feature
         public FailedMessagesFeature()
         {
             EnableByDefault();
-            RegisterStartupTask<WireUpFailedMessageNotifications>();
         }
 
         protected override void Setup(FeatureConfigurationContext context)
         {
             context.Container.ConfigureComponent<FailedMessageViewIndexNotifications>(DependencyLifecycle.SingleInstance);
             context.Container.ConfigureComponent<DetectSuccessfullRetriesEnricher>(DependencyLifecycle.SingleInstance);
+
+            context.RegisterStartupTask(b => b.Build<WireUpFailedMessageNotifications>());
```

Comment:
I think that startup tasks are not added to the container unless we explicitly add them

Reply:
I guess we'll figure that out easily enough once the tests start